packages/example-vue/package.json

```diff
@@ -18,8 +18,8 @@
   },
   "devDependencies": {
     "@playwright/test": "^1.30.0",
-    "@vitejs/plugin-vue": "^4.0.0",
-    "msw": "1.0.0",
+    "@vitejs/plugin-vue": "^4.5.0",
+    "msw": "2.0.8",
```

Comment:
Maybe you can do 1 more update -> `2.0.10` before merging

Reply:
Great idea, unfortunately I had already merged this PR. I've updated msw to `2.0.10` in a separate commit though: https://github.com/valendres/playwright-msw/commit/0740611f0973b4782784c826244c96938cb651d9